bokehjs/src/lib/models/widgets/tables/cell_formatters.ts

```diff
@@ -81,6 +81,63 @@ export class StringFormatter extends CellFormatter {
   }
 }
 
+export namespace ScientificFormatter {
+  export type Attrs = p.AttrsOf<Props>
+
+  export type Props = StringFormatter.Props & {
+    precision: p.Property<number | 10>
```

Comment:
` | 10` doesn't seem necessary to me. What's the reasoning behind this?

Reply:
yes, good point, I assumed it was the default value when copying it from BasicTickFormatter which mentions precision: p.Property<number | "auto">